community_tasks/swiss_legal_evals.py

```diff
@@ -0,0 +1,402 @@
+        ),
+        "paragraph_level": LevelConfig(
+            name="paragraph_level",
+            text_col_name="paragraphText",
+            prompt_prefix="Consider the following Swiss federal law paragraph",
+            metadata_cols=["abbreviation", "url", "dateApplicability", "rsNr"],
+        ),
+    },
+)
+
+SwissSupremeCourtPressReleaseTranslations = DatasetConfig(
+    name="sscprt",
+    hf_repo="joelniklaus/SwissSupremeCourtPressReleaseTranslations",
+    languages=["de", "fr", "it"],
+    subsets={
+        "press_release": LevelConfig(
+            name="press_release",
+            text_col_name="text",
+            prompt_prefix="Consider the following Swiss Supreme Court press release",
+            metadata_cols=["filename"],
+        )
+    },
+)
+
+
+def create_prompt_fn(level_config: LevelConfig, src_lang: str, target_lang: str):
+    """
+    Create a prompt function for a given level configuration.
+    """
+    text_col = level_config.text_col_name
+    src_text_col = f"{src_lang}_{text_col}"
+    target_text_col = f"{target_lang}_{text_col}"
+
+    def prompt_fn(line: dict, task_name: str = None):
+        custom_query = f"{level_config.prompt_prefix}: {line[src_text_col]}\nTranslate from {src_lang} to {target_lang}.\nTranslation: "
+
+        return Doc(
+            task_name=task_name,
+            query=custom_query,
+            choices=[str(line[target_text_col])],
+            gold_index=0,
+            specific={
+                **{col: line[col] for col in level_config.metadata_cols},
+                "question": custom_query,
+            },
+        )
+
+    return prompt_fn
+
+
+class TranslationTask(LightevalTaskConfig):
+    def __init__(
+        self,
+        dataset_config: DatasetConfig,
+        level_name: str,
+        src_lang: str,
+        target_lang: str,
+    ):
+        super().__init__(
+            name=f"{dataset_config.name}-{level_name}:{src_lang}-{target_lang}",
+            suite=["community"],
+            prompt_function=create_prompt_fn(
+                dataset_config.subsets[level_name], src_lang, target_lang
+            ),
+            hf_repo=dataset_config.hf_repo,
+            hf_subset=level_name,
+            hf_filter=None,
+            hf_avail_splits=["train", "validation", "test"],
+            evaluation_splits=["test"],  # ["validation", "test"],
+            few_shots_split="validation",
+            few_shots_select=None,
+            generation_size=10,
+            metric=[
+                Metrics.bleu,
+                Metrics.bleu_1,
+                Metrics.bleu_4,
+                Metrics.chrf,
+                Metrics.ter,
+                bert_score(model_type="xlm-roberta-large"),
+                bleurt(model_size="tiny", seq_len=512),
+                freeform_gpt_judge(judge_model_name="gpt-4o"),
+                # freeform_flow_judge(), # TODO: Needs to be tested on GPU machine
```

Comment:
Should we wait for these tests before merging your PR?

Reply:
Yes, I would say so. Just wanted to bring it up already now to look at the other parts.

Reply:
I removed it now, so this is not blocking anymore.